pulseapi/entries/migrations/0009_remove_entry_is_approved.py

```diff
@@ -0,0 +1,19 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.10.3 on 2017-05-25 23:07
+from __future__ import unicode_literals
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('entries', '0008_entry_moderation_state'),
+    ]
+
+    operations = [
+        migrations.RemoveField(
```

Comment:
after all the data's been moved over to use moderation_state, the `is_approved` flag is now a soft prop based on the moderation state, and is no longer needed as Boolean field in the model